.github/actions/check-copyright/action.yaml

```diff
@@ -1,5 +1,5 @@
 name: Check Copyright
-description: Common steps to check the copyright of our components
+description: Common steps to perform a code scan on a components
```

Comment:
Revert

Reply:
Yeah sorry updated the wrong one, more coffee